phpBB/phpbb/storage/storage.php

```diff
@@ -136,6 +163,8 @@ public function delete($path)
 	public function rename($path_orig, $path_dest)
 	{
 		$this->get_adapter()->rename($path_orig, $path_dest);
+		$this->untrack_file($path_orig);
```

Comment:
Sounds a bit inefficient to me. Could we maybe add another method like `update_file($path_orig, $path_dest)`?

Reply:
Yes